### DISCUSSION TITLE
Add white-space: nowrap;

### DISCUSSION THREAD

**Discussion by Jem20** [anthony1x6000/ROBLOX2016stylus#35] [Ideas]
You can add this because when you search for a game or even on the homepage the title is cut off and having 3 "..." would make it look better imo.

``.game-card-name, .game-name-title {
        white-space: nowrap;
    }``

**Comment by anthony1x6000** [comment]
I might make this an optional thing. It does look nice but sometimes it cuts off the last letter of the game title.
